### DISCUSSION TITLE
Threads

### DISCUSSION THREAD

**Discussion by TheNitram21** [discord-jda/JDA#1743] [General]
Discord just released Threads.

A few hours ago they released a YouTube video about them. Currently, only like 10% of all discord servers have threads, but they say that they want to release them to every server at the end of the week. Will threads be added to JDA when they are released to all servers? I could imagine some easy code like `Message#createThread()`, and I would really appreciate a pull request adding them in the near future.

**Comment by TheCipher01** [comment]
They're in a draft state atm #1613
